### PR TITLE
Ije 71

### PR DESCRIPTION
**Pull request by phillip-laTRN**
I was actually working on the routing for our application this morning. Found out that @fahadahmed @fhdamd was doing the same. This branch contains the merging of my routing with Fahad's.

### CODE REVIEW

File: packages/frontend/src/components/VerificationMessage.tsx [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-            <h2 data-testid="signup-heading">Sign up to Infosys Goodies</h2>
-            <h2 data-testid="almost-there-message">Almost There!</h2>
-            <h3 data-testid="instruction-message-1">We've sent a verification link to your email!</h3>
-            <h3 data-testid="instruction-message-2">Click on the link to finish creating your account.</h3>
-        </div>
-    );
-};
+const VerificationMessage = () =>
+    <div>
+        <h2 data-testid="signup-heading">Sign up to Infosys Goodies</h2>
```
Comment (fahadahmed) [inline packages/frontend/src/components/VerificationMessage.tsx:6]: Would like the tests to be written without the usage of `data-testid`. This leads to dirty components.

**Review by fahadahmed — CHANGES_REQUESTED** [review]
@phillip-laTRN can you please address the feedback when you get a chance?

File: packages/frontend/src/components/SignupBankQuery.tsx [hunk 2]
```diff
@@ -1,14 +1,12 @@
-import React from "react";
+import * as React from "react";
+import { Link } from "react-router-dom";
 
-const SignupBankQuery = () => {
```
[the remark this answers is not in the document]
Comment (phillip-laTRN) [reply packages/frontend/src/components/SignupBankQuery.tsx:3]: I agree that the naming can be ambiguous. The "Query" in "SignupBankQuery" means that this component is querying/asking the user whether they want to proceed to add additional information (bank information, in this case) during their new account registration process. It's not a form as the actual form for filling in this information is in the SignupBank component. 

Is there another name that you suggest for the SignupBankQuery component?

[the remark this answers is not in the document]
Comment (fahadahmed) [reply packages/frontend/src/components/SignupBankQuery.tsx:3]: Maybe then use `SignupBankCheck`.

**Comment by phillip-laTRN** [comment]
@fahadahmed @fhdamd I have made the requested changes to this branch. Please have a look

**Review by fhdamd — APPROVED** [review]

**Comment by fhdamd** [comment]
Hi @phillip-laTRN , can you please resolve this merge conflicts?

**Comment by phillip-laTRN** [comment]
@fahadahmed @fhdamd I have solved the merge conflicts. Please have a look and accept if all is good :)

**Review by fhdamd — APPROVED** [review]

**Review by fahadahmed — APPROVED** [review]

**Pull request merged**